CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](http://semver.org/).
 In order to read more about upgrading and BC breaks have a look at the [UPGRADE Document](UPGRADE.md).
 
+## 1.1.3 (27. May 2019)
+
++ Added deduplicate flag to be able to send a mail more than once to a recipient in the same campaign
```

Comment:
This is a very fresh changelog file, so there are no good example containing yet, my fault! 

Please refer the changelog entry to your Pull request like:

```
- [#1](https://github.com/luyadev/luya-mailjet/pull/1) Added deduplicate flag to be able to send a mail more than once to a recipient in the same campaign and Added CustomCampaign and CustomId to provide additional tagging ability
```

Reply:
Ok, I've changed this.